### ISSUE TITLE
SyncTrayzor crash with Syncthing 0.13-beta.2

### ISSUE THREAD

**Issue by lpoujol** [canton7/SyncTrayzor#250]
Hi.

I manualy updated syncthing from 0.12.21 to 0.13-beta.2 to try it. But now SyncTrayzor crashes when it run sycthing :(

StackTrace given : 
```
> Version: 1.1.8.0; Variant: Installed; Arch: Amd64
> Path: D:\Programmes\SyncTrayzor\SyncTrayzor.exe
> System.NullReferenceException: La référence d'objet n'est pas définie à une instance d'un objet.
>    à SyncTrayzor.Syncthing.Folders.FolderIgnores..ctor(List`1 ignores, List`1 patterns) dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\Folders\FolderIgnores.cs:ligne 23
>    à SyncTrayzor.Syncthing.Folders.SyncthingFolderManager.<>c__DisplayClass29_0.<<FetchFoldersAsync>b__2>d.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\Folders\SyncthingFolderManager.cs:ligne 180
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Syncthing.Folders.SyncthingFolderManager.<FetchFoldersAsync>d__29.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\Folders\SyncthingFolderManager.cs:ligne 187
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Syncthing.Folders.SyncthingFolderManager.<LoadFoldersAsync>d__27.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\Folders\SyncthingFolderManager.cs:ligne 108
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Syncthing.SyncthingManager.<LoadConfigDataAsync>d__137.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\SyncthingManager.cs:ligne 497
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Syncthing.SyncthingManager.<LoadStartupDataAsync>d__136.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\SyncthingManager.cs:ligne 475
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Syncthing.SyncthingManager.<StartClientAsync>d__130.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\SyncthingManager.cs:ligne 367
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Syncthing.SyncthingManager.<StartAsync>d__120.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Syncthing\SyncthingManager.cs:ligne 195
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Utils.SafeSyncthingExtensions.<StartWithErrorDialogAsync>d__0.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Utils\SafeSyncthingExtensions.cs:ligne 17
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
>    à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    à SyncTrayzor.Pages.ShellViewModel.<Start>d__39.MoveNext() dans C:\projects\synctrayzor\src\SyncTrayzor\Pages\ShellViewModel.cs:ligne 89
> --- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
>    à System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
>    à System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
>    à System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
>
```

**Comment by canton7** [comment]
I don't support the beta yet unfortunately. I'll look at adding support this week

**Comment by canton7** [comment]
This build should do it: https://ci.appveyor.com/project/canton7/synctrayzor/build/1.1.8.334/artifacts

**Comment by canton7** [comment]
Fixed in v1.1.9
